docs/config/cluster.rst

```diff
@@ -1156,3 +1177,7 @@ Metadata gateway
   because you only want the cluster state to be recovered once all nodes are
   started. However, the value must be bigger than the half of the expected
   number of nodes in the cluster.
+
```

Comment:
minor: delete this line

the standard way to separate the links is with two empty lines